src/components/calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display">0</div>
```

Comment:
- [ ] Please adhere 🙏🏻 to the project requirements according to which you should add at least **one** child component inside the `Calculator` component. But it seems like you just added the `JSX` code and forgot to add the child component. Kindly remove the display `JSX` code from your code and make a separate component like `Display` and then import the component in this component. Basically, if you have a more modular code in your application it will be far easier for you to resolve the error or bugs and make your code more clear and readable.

## Requirements

![image](https://github.com/balatstar/math-magicians/assets/66476329/4e7baf93-0065-49fb-9a98-38143e1fe7b7)